webrender/res/ps_border_corner.glsl

```diff
@@ -322,7 +322,7 @@ void main(void) {
     vec2 local_pos = vLocalPos;
 #endif
 
-    alpha = min(alpha, do_clip());
+    alpha *= do_clip();
```

Comment:
nit: in shaders like this one, we can simply start with `float alpha = do_clip();`, remove that awkward `alpha = 0.0;` and end up with fewer lines ;)

Reply:
As discussed on IRC, this won't work because then `init_transform_fs` would overwrite the value from `do_clip()`.